### PR TITLE
Update readme: setting up a database

### PR DESCRIPTION
**Pull request by andreibancioiu**
- How to set up an observer database, in order to support historical balances lookup.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,20 +20,35 @@ Follow the official documentation to setup an observer:
...
+   FullArchive = true
 ```
 
-Furthermore, enable the **database lookup extensions** in `config/config.toml`, as follows:
+Furthermore, adjust `config/config.toml`, as follows:
 
 ```
+[GeneralSettings]
+    StartInEpochEnabled = false
+
```
Comment (bogdan-rosianu) [inline README.md:36]: add `...` between sections? one might think these lines are next to each other

File: README.md [hunk 2]
```diff
@@ -62,6 +81,102 @@ Or, in order to start using the `offline` mode:
 --port=9092 --offline
 ```
 
+## Setup a database
+
+In order to support historical balances lookup, Rosetta has to connect to an Observer whose database contains _non-pruned accounts tries_. Such databases can be re-built locally or downloaded from the Internet.
```
Comment (bogdan-rosianu) [inline README.md:86]: downloaded from Elrond public archive, whose link can be obtained from Elrond Developers tg ?

File: README.md [hunk 3]
```diff
@@ -62,6 +81,102 @@ Or, in order to start using the `offline` mode:
...
+
+#### `config.toml`
+```
+[StoragePruning]
+    AccountsTrieCleanOldEpochsData = false
+    NumEpochsToKeep = 128 # a desired history length
+
+[StateTriesConfig]
+    AccountsStatePruningEnabled = false
+
```
Comment (bogdan-rosianu) [inline README.md:100]: add `...` between sections

File: README.md (hunk 1 above)
Comment (andreibancioiu) [reply README.md:36]: Added.

File: README.md (hunk 3 above)
Comment (andreibancioiu) [reply README.md:100]: Added.

File: README.md (hunk 2 above)
Comment (andreibancioiu) [reply README.md]: Adjusted.

**Review by bogdan-rosianu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**